test/fixtures/rules/std_rule_cases/L058.yml

```
@@ -0,0 +1,189 @@
+rule: L058
+
+test_pass_1:
+  # The nested CASE is under a "WHEN", not an "ELSE".
+  pass_str: |
+    SELECT
+        c1,
+        CASE
+            WHEN species = 'Rat' THEN
+              CASE
+                  WHEN species = 'Dog' THEN 'Woof'
+              END
```

I don't understand this example? Instead of returning 'squeak' we go into another case (also for species, but a different one, so this can never be true). I get it's just an example but should we pick a better one that makes sense?

```suggestion
            WHEN species = 'Rat' THEN
              CASE
                  WHEN colour = 'Black' THEN 'Growl'
                  WHEN colour = 'Grey' THEN 'Squeak'
              END
```

---

Ha, great point! I was in "robotic example mode." 🤪